src/lang/es-419/interfaces.json

```
@@ -182,10 +182,10 @@
 			"filters_comment": "Qué filtros usar",
 			"accept": "Aceptar los Tipos de Archivo",
 			"accept_comment": "Estableces una lista de tipos MIME que puedan ser elegidos",
-			"allow_delete": "Allow Delete",
-			"allow_delete_comment": "Allows the user to permanently delete the related file through the file interface",
-			"crop_preview": "Crop File Preview",
-			"crop_preview_comment": "Crops the file preview to fill the available space."
+			"allow_delete": "Permitir Eliminar",
+			"allow_delete_comment": "Permite al usuario eliminar permanentemente el archivo relacionado a través de la interfaz de archivos"
```

You forgot a `,` at the end here 🙂 

```suggestion
			"allow_delete_comment": "Permite al usuario eliminar permanentemente el archivo relacionado a través de la interfaz de archivos",
```